[All Hosts] (browsers) restore info about browser switch command and … (#3041)
OfficeDev/office-js-docs-pr: docs/testing/debug-add-ins-using-devtools-edge-legacy.md

If your add-in uses the Office Dialog API, the dialog runs in a separate process from the task pane (if any) and the tools must attach to that process. Follow these steps.

1. Run the add-in and the tools.
1. Open the dialog and then select the **Refresh** button in the tools. The dialog process is shown. Its name comes from the `<title>` element in the HTML file that is open in the dialog.
1. Select the process to open it and debug just as described in the section [Debug a task pane add-in using Microsoft Edge DevTools Preview](#debug-a-task-pane-add-in-using-microsoft-edge-devtools-preview).

   :::image type="content" source="../images/edge-devtools-with-add-in-and-dialog-processes.png" alt-text="Screenshot of Edge DevTools showing a process named My Dialog.":::

## Switch to the Edge Legacy webview

> 

Reviewer: The content in this section seems to be the same (or pretty similar) to that in the other files. Any reason not to use an include?

Author: It is using includes. But the headings and opening paragraph need to be different. One article is about Edge Legacy and others are about IE.

> There are two ways to switch the Edge Legacy webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Edge Legacy by default. We recommend the first method. But you should use the second in the following scenarios:

Reviewer: ```suggestion
There are two ways to switch the Edge Legacy webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Edge Legacy by default. We recommend the first method. But you should use the second in the following scenarios.
```

> - Your project was developed with Visual Studio and IIS. It isn't nodejs-based.

Reviewer: ```suggestion
- Your project was developed with Visual Studio and IIS. It isn't node.js-based.
```

- You want to be absolutely robust in your testing.
- If for any reason the command line tool doesn't work.

> ### Switch on the command line

Reviewer: I'm not sure about saying "on" - it sounds like turning on the command line which I don't think is the intention. Maybe "via" or "using"...

Here and other instances

```suggestion
### Switch via the command line
```
